gcore/src/exec.rs

```diff
@@ -107,9 +106,9 @@ pub fn block_timestamp() -> u64 {
 /// }
 /// ```
 pub fn reply_deposit(message_id: MessageId, amount: u64) -> Result<()> {
-    let mut len = 0u32;
-    unsafe { gsys::gr_reply_deposit(message_id.as_ptr(), amount, &mut len as *mut u32) };
-    SyscallError(len).into_result()
+    let mut error_code = 0u32;
```

Comment:
Shouldn't it be `u32::MAX`?

`SyscallError` thinks that the result is successful only if it's `u32::MAX`. Also `reply_deposit` doesn't mutate value to `u32::MAX` if the execution was successful, as far as I see.

Reply:
No, 0 is used as a success code everywhere

Reply:
Shouldn't some test fail if it is incorrect?

Reply:
didn't recognize that
https://github.com/gear-tech/gear/blob/623adb02b5d698b9b2d3597f70fa4cf9cb366935/gsys/src/lib.rs#L187
everything is ok